tests/cypress/accessibility/axe-concept.cy.js

```diff
@@ -1,13 +1,16 @@
 // When you start creating tests, remove the following comment(s).
-// import { accessibilityTestRunner } from '../support/accessibility.js'
-// import 'cypress-axe';
+import { accessibilityTestRunner } from '../support/accessibility.js'
+import 'cypress-axe'
 
 /* If you want the test to be skipped, add a skip command after the describe part:
     - test enabled: describe('Check accessibility of ...
     - test to be skipped: describe.skip('Check accessibility of ... */
-describe.skip('Check accessibility of the concept page', () => {
-  it('no-op test', () => {})
+describe('Check accessibility of the concept page', () => {
+  before(() => {
+    cy.visit('yso/fi/page/p4623')
+    cy.injectAxe()
+  })
 
   // When you start creating tests, remove the following comment(s).
```

Comment:
I think this comment line should be removed.

Reply:
Removed